src/client.rs

```diff
@@ -32,16 +32,23 @@ pub struct Client {
 
 impl Client {
     /// Constructs a new `Client`.
+    #[cfg(feature = "default-tls")]
     pub fn new() -> ::Result<Client> {
         let mut client = try!(new_hyper_client());
         client.set_redirect_policy(::hyper::client::RedirectPolicy::FollowNone);
-        Ok(Client {
+        Ok(Client::with_hyper_client(client))
+    }
+
+    /// Constructs a new `Client` with the provided `::hyper::Client` without
+    /// altering any settings.
```

Comment:
I guess the provided client should be left alone, then. I wouldn't want an extra `_raw` method. Perhaps best to point out that reqwest won't adjsut the Client, even though it normally would alter the defaults in the `new` constructor.